saos-webapp/src/main/webapp/WEB-INF/tags/details/correctionJudgment.tag

```diff
@@ -0,0 +1,32 @@
+<%@ include file="/WEB-INF/view/common/taglibs.jsp" %>
+
+<%@ attribute name="correction" required="true" rtexprvalue="true" type="pl.edu.icm.saos.persistence.correction.model.JudgmentCorrection" %>
+
+
+<c:choose>
+	<c:when test="${correction.correctedProperty == 'JUDGMENT_TYPE'}" >
+
+		<c:set var="oldValue" value="" />
+		<c:forEach var="code" items="${fn:split(correction.oldValue, ', ')}" varStatus="status">
+			<spring:message var="springMessage" code="pl.edu.icm.saos.persistence.model.Judgment$JudgmentType.${code}" />
```

Comment:
Their judgmentTypes may be anything, they are not the same as our ones, so you shouldn't use JudgmentType properties.